go.mod

```diff
@@ -8,6 +8,9 @@ require (
 	github.com/adlio/schema v1.1.14
 	github.com/btcsuite/btcd v0.22.0-beta
 	github.com/btcsuite/btcutil v1.0.3-0.20201208143702-a53e38424cce
+	github.com/facebookgo/ensure v0.0.0-20160127193407-b4ab57deab51 // indirect
+	github.com/facebookgo/stack v0.0.0-20160209184415-751773369052 // indirect
+	github.com/facebookgo/subset v0.0.0-20150612182917-8dac2c3c4870 // indirect
```

Comment:
huh?

Reply:
```sh
tm-db % go mod why github.com/facebookgo/ensure
# github.com/facebookgo/ensure
github.com/tendermint/tm-db
github.com/tecbot/gorocksdb
github.com/tecbot/gorocksdb.test
github.com/facebookgo/ensure
```

Reply:
What I don't understand is why Dependabot has three PRs open against this upgrade.

Reply:
different branches!

Reply:
ok thanks for the context

Reply:
> different branches!

Oh yes, of course.